deploy.sh

```diff
@@ -149,20 +152,20 @@ function start_link_checker_processes {
     start_new_process "Creating Rabbit MQ user and vhost for celery"
     cd $APP_DIR
 
-    until docker-compose exec -T rabbitmq rabbitmqctl start_app; do
+    until docker-compose --project-name=$ENV exec -T rabbitmq rabbitmqctl start_app; do
         log "Rabbit is unavailable - sleeping"
         sleep 10
     done
 
-    docker-compose exec -T rabbitmq rabbitmqctl add_user di_website $RABBITMQ_PASSWORD
-    docker-compose exec -T rabbitmq rabbitmqctl add_vhost myvhost
-    docker-compose exec -T rabbitmq rabbitmqctl set_user_tags di_website di_website
-    docker-compose exec -T rabbitmq rabbitmqctl set_permissions -p myvhost di_website ".*" ".*" ".*"
+    docker-compose --project-name=$ENV exec -T rabbitmq rabbitmqctl add_user test-user $RABBITMQ_PASSWORD
+    docker-compose --project-name=$ENV exec -T rabbitmq rabbitmqctl add_vhost myvhost
+    docker-compose --project-name=$ENV exec -T rabbitmq rabbitmqctl set_user_tags test-user test-user
+    docker-compose --project-name=$ENV exec -T rabbitmq rabbitmqctl set_permissions -p myvhost test-user ".*" ".*" ".*"
```

Comment:
Wasn't this test-user change meant for your test deployment? If so, this won't work on staging or live ... it's imperative that you keep your test server in sync with the others.